src/client/mapblock_mesh.cpp

```diff
@@ -30,6 +30,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/directiontables.h"
 #include "client/meshgen/collector.h"
 #include "client/renderingengine.h"
+#include "client/localplayer.h"
```

Comment:
why localplayer is included here ? we should not have player inclusion on mapblock_mesh, which is close to render and has nothing to do with player session/object